knpu/ep3-webpack/require-sweet-alert.md

```diff
@@ -28,7 +28,7 @@ library we're using.
 So go get it peeps! Find your terminal and run:
 
 ```terminal
-yarn add sweetalert2 --dev
```

Comment:
Let's add a note here (mostly because we'll need to add the note to the video):

> To follow the tutorial exactly, install the exact same version with `yarn add sweetalert2@6.6.6 --dev`